File: code-coaches/medium/pig-latin/PigLatin.cs
using System;

public class PigLatin
{
    static string ToPigLatin(string sentence)
    {
        string[] words = sentence.Split(' ');
        string result = "";

        foreach (string word in words)
        {
            string firstLetter = word.Substring(0, 1);
            string restOfWord = word.Substring(1);
            result += restOfWord + firstLetter + "ay ";
        }

        return result.Trim();
    }

    static void Main(string[] args)
    {
        string sentence = Console.ReadLine();
        Console.WriteLine(ToPigLatin(sentence));
    }
}

File: code-coaches/medium/pig-latin/PigLatin.java
import java.util.Scanner;

public class PigLatin {

    static String toPigLatin(String sentence) {
        String[] words = sentence.split(" ");
        String result = "";

        for (String word : words) {
            String firstLetter = word.substring(0, 1);
            String restOfWord = word.substring(1);
            result += restOfWord + firstLetter + "ay ";
        }

        return result.trim();
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String sentence = input.nextLine();

		System.out.println(toPigLatin(sentence));

		input.close();
	}

}